Lifecycle.md

```diff
@@ -0,0 +1,279 @@
+# Planetary Software Organization Project and Working Group Lifecycle
+
+## Definitions
+
+The Planetary Software Organization hosts several Top-Level Projects
+(TLPs). These projects are autonomous from each other and governed
+by their own TC (Technical Committee) and chartered by the Planetary
+Software TSC.  Typically, TLPs are software projects (but not always)
+and are typically long-lived and open-ended.
+
+The Planetary Software Organization also hosts several Top-Level Working
+Groups (TLWGs).  TLWGs are typically formed for a specific purpose,
+and when that purpose is achieved, the TLWG will stand down.
+
+TLPs are free to create their own Working Groups which are autonomous groups
+collaborating to fulfill a set of responsibilities. Working Groups are
+eventually chartered by their project TC.
+
+```
+ TSC
+  |
+  |-- Project A TC (Chartered By TSC)
+  |       |-- Working Group (Chartered By Project A TC)
+  |
+  |-- Project B TC (Chartered By TSC)
+  |       |-- Working Group (Chartered By Project B TC)
+  |
+  |-- Working Group X (Chartered by TSC)
+  |-- Working Group Y (Chartered by TSC)
+```
+
+Both TLPs and TLWGs may elect a representative to the TSC. TLPs and WGs
```

Comment:
Do we want this to be optional?

Reply:
This is actually a redundant statement, so I'm removing it here.  You have a good point.  I'd rather let it stand for now (this is how it was written before) and then we bring it up separately, rather than buried in this update, as it seems a meatier topic.